src/transformPolyfillRequirePlugin.js

```diff
@@ -0,0 +1,75 @@
+const polyfillSource = "babel-polyfill";
```

Comment:
Oh I guess it should probably work on `import "core-js"` too..

and right now it's weird that it would be change  `import "babel-polyfill"` to "core-js" (since babel-polyfill doesn't have any generated modules atm anyway..)